docs/cimpl6ToolingReference.md

```diff
@@ -342,83 +343,69 @@ The Content Profile (CP) specifies the profiles to be included in the IG. As exp
 
-The CP provides several different types of information about the IG:
+The CP provides several types of information about the IG:
 
-* The list of classes (profiles) to be included in the IG,
-* The elements in these classes to be marked as [MustSupport](#Specifying-must-support-elements), designated with the `MS` tag, and
-* A list of classes _not_ to be profiled, designated with the [No-Profile](#specifying-no-profile-elements) `NP` tag.
+* The list of Entries (or entire namespaces) whose profiles are to be included in the IG,
+* A list of classes (or entire namespaces) _not_ to be profiled, and
+* Elements in the profiled classes to be marked as [MustSupport](#Specifying-must-support-elements).
 
-The CP file is interpreted as follows:
+#### Specifying Profiles to Include in the IG
 
-* The classes listed in the CP are included as profiles in the IG, with the exception of those classes marked as  (`NP`).
-* A class _not_ explicitly listed in the CP _may be included_ in the IG, if the class (1) is referenced directly or indirectly by a class included in the IG, (2) is not tagged as a No-Profile class, and (3) has a [differential](https://www.hl7.org/fhir/structuredefinition-definitions.html), relative to the resource it is mapped to.
+As illustrated above, the FHIR IG may include models from multiple namespaces. The first job of the CP is to specify the profiles that should be presented in the IG, out of the available `Entry` models in class files in the _spec_ directory.
 
-The syntax of a Content Profile file is:
+To include the profile of an `Entry` in the IG, list the class name in the CP. The list must be arranged by namespace, as shown in [the example below](#content-profile-example). To include all Entries in a namespace, mark the namespace with a wildcard `*`. We refer to any `Entry` explicitly listed for inclusion as "primary" profiles.
 
-```
-Grammar:  ContentProfile 1.0
-
-Namespace:  <namespace-1>
-    <Entry-1>:
-        <Element-1> MS
-        <Element-2> MS
-        ...
-    <Entry-2>: NP
-    ...
-Namespace:  <namespace-2>
-    <Entry-1>:
-        <Element-1> MS
-        <Element-2> MS
-        ...
-    <Entry-2>: NP
-    ...
-```
+> **Note**: Only `Entries` result in FHIR profiles. `Groups` and `Elements` result in extensions. Inclusion of extensions in the IG is automatic, and cannot be specified in the CP.
 
-For example:
+#### Specifying No-Profile Classes
 
-```
-    Grammar: ContentProfile 1.0
+When you specify an `Entry` to be included in an IG, it may indirectly pull in other `Entries`. For example, an Observation references other classes including Patient, Specimen, Device, and Practitioner. In turn, Specimen could reference a Procedure, Device an Organization, etc, until a large number of classes are involved. By default, if there is a CIMPL model for a referenced class, SHR-CLI will create a profile for that class, and include it in the IG. We refer to these dependent profiles as _supporting_ profiles.
```

Comment:
Are Observation, Patient, Specimen, Device and Practitioner references to FHIR resources. If yes, bold.

Reply:
ok